Src/WitsmlExplorer.Frontend/components/ContextMenus/LogCurveInfoContextMenu.tsx

```diff
@@ -379,35 +420,73 @@ const LogCurveInfoContextMenu = (
             )}
           </Typography>
         </MenuItem>,
-        <MenuItem
-          key={"setPriority"}
-          onClick={() =>
-            onlyPrioritizedCurvesAreChecked
-              ? onClickRemovePriority()
-              : onClickSetPriority()
-          }
-        >
-          <StyledIcon
-            name={
+        <NestedMenuItem key={"logPriorityCurves"} label={"Log Priority Curves"}>
+          <MenuItem
+            key={"setPriority"}
+            onClick={() =>
               onlyPrioritizedCurvesAreChecked
-                ? "favoriteFilled"
-                : "favoriteOutlined"
+                ? onClickRemovePriority(false)
+                : onClickSetPriority(false)
             }
-            color={colors.interactive.primaryResting}
-          />
-          <Typography color={"primary"}>
-            {onlyPrioritizedCurvesAreChecked
-              ? "Remove Priority"
-              : "Set Priority"}
-          </Typography>
-        </MenuItem>,
-        <MenuItem key={"editPriority"} onClick={onClickEditPriority}>
-          <StyledIcon
-            name="favoriteOutlined"
-            color={colors.interactive.primaryResting}
-          />
-          <Typography color={"primary"}>Edit Priority</Typography>
-        </MenuItem>,
+          >
+            <StyledIcon
+              name={
+                onlyPrioritizedCurvesAreChecked
+                  ? "favoriteFilled"
+                  : "favoriteOutlined"
+              }
+              color={colors.interactive.primaryResting}
+            />
+            <Typography color={"primary"}>
+              {onlyPrioritizedCurvesAreChecked
+                ? "Remove Local Priority"
+                : "Set Local Priority"}
+            </Typography>
+          </MenuItem>
+          ,
```

Comment:
Remove the commas from within the _NestedMenuItem_ as they cause the row height problem mentioned before.

Reply:
done